frontend/src/components/feed/Feed.js

```diff
@@ -33,7 +34,8 @@ const Feed = ({ navigate }) => {
           <h2>Posts</h2>
             <button onClick={logout}>
               Logout
-            </button>
+          </button>
```

Comment:
absolutely nit picking here but be nice to keep the indentation of the button closing tag